rpcs3/cmake_modules/ConfigureCompiler.cmake

```diff
@@ -9,6 +9,8 @@ if(MSVC)
 	#TODO: Some of these could be cleaned up
 	add_compile_options(/wd4805) # Comparing boolean and int
 	add_compile_options(/wd4804) # Using integer operators with booleans
+	add_compile_options(/wd4200) # Zero-sized array in struct/union
```

Comment:
wtf where have seen a zero sized array?

Reply:
It's not being used in rpcs3 but libusb uses them a lot.